integration-tests/metaChecker.go

```diff
@@ -67,7 +73,7 @@ func checkMetaSCRs(apiSCRs []gjson.Result, scrs map[string]*firehose.SCRInfo) er
 			return err
 		}
 
-		scrFromProtocol, found := scrs[string(hashBytes)]
+		scrFromProtocol, found := scrs[hash]
```

Comment:
perhaps this check can be moved before L71, thus in case of !found, the DecodeString won't be done anymore

Reply:
RIght, changed as suggested